server/routes/objects.js

```diff
@@ -197,22 +192,20 @@ const handleAllUpload = async (
             resolve(
               handleUpload(
                 bucketName,
+                user,
                 minioPath,
                 filePath,
                 obj,
                 tempDirPath,
                 fileName,
                 token,
-                user,
-                format,
-                fileId,
               ),
             ),
           );
         });
       } catch (error) {
         // Handle any errors that occurred during handleUpload
-        console.error('Error during handleUpload:', error);
+        log.error('Error during handleUpload:', error);
       }
       next();
```

Comment:
The error handling for child process stderr should log the error using the logger instead of console.error. This ensures consistent logging throughout the application.
```diff
-            console.error(`stderr: ${data}`);
+            log.error(`stderr: ${data}`);
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*